vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/crawler/transformer/AbstractFetchingUrlsTableTransformer.java

```diff
@@ -101,6 +103,24 @@ private String build(Set<String> urls, TableProperties properties)
                 .buildExamplesTableFromColumns(List.of(columnName), List.of(urlsList), properties);
     }
 
+    protected URI getMainApplicationPageUri(TableProperties properties)
+    {
+        URI uri = Optional.ofNullable(properties.getProperties().getProperty("mainPageUrl"))
+                          .map(UriUtils::createUri)
+                          .orElse(mainPageUrl);
+
+        if (uri == null)
+        {
+            uri = webApplicationConfiguration.getMainApplicationPageUrl();
+            logger.atWarn().addArgument("web-application.main-page-url")
+                           .log("The use of {} property for setting of main page for crawling is deprecated and will "
+                                   + "be removed in VIVIDUS 0.7.0, pelase see official documentation for corresponding"
+                                   + " replacements.");
```

Comment:
it's better to provide replacement options right, instead of asking users to search for the documentation

Reply:
well its easy to describe transformer parameter but when it comes to property name we will have it pass from child classes making it clumsy